src/simplewallet/simplewallet.cpp

```diff
@@ -2227,6 +2226,7 @@ bool simple_wallet::transfer_main(int transfer_type, const std::vector<std::stri
     else while (!ptx_vector.empty())
     {
       auto & ptx = ptx_vector.back();
+      ptx.is_onetime = is_onetime;
```

Comment:
Ah, I see what you mean. Then I guess I need to put `ptx.is_onetime = is_onetime;` into the end of `wallet2::transfer<T>()` and `wallet2::transfer_selected()` and `wallet2::transfer_selected_rct()`, correct?

BTW, overloads to `wallet2::transfer()` without args `cryptonote::transaction tx` and `pending_tx ptx` seem to be not used anywhere. Aren't they redundant?